cmd/up/up.go

```diff
@@ -960,6 +974,24 @@ func (up *upContext) getInteractive() bool {
 	return false
 }
 
+func (up *upContext) getInsufficientSpaceError(err error) error {
+	if up.Dev.PersistentVolumeEnabled() {
+		return errors.UserError{
+			E: err,
+			Hint: `Okteto volume is full
+    Increase your persistent volume size, run 'okteto down -v' and try 'okteto up' again
```

Comment:
are we using punctuation in the multi-line error messages? I think we should, since they are more explanation that error message.

Reply:
We are using both, I am trying to unify that here. Would you add punctuation on every line?

Reply:
Yes.

Reply:
I'd use punctuation in the hint, and leave the error as is. specially in these cases where the hint is more of a paragraph.